### ISSUE TITLE
安装elk

### ISSUE THREAD

**Issue by mchehnag** [elastic/windows-installers#321]
<

![image](https://user-images.githubusercontent.com/32324101/61689404-b745a480-ad59-11e9-8d50-1bfa8ea6ef86.png)

**Comment by mchehnag** [comment]
这是什么问题 希望能即使解决

**Comment by russcam** [comment]
Hi @mchehnag , 

Please attach the MSI log file for the failed installation; a log file for the installation process can be found within the `%TEMP%` directory, with a randomly generated name adhering to the format `MSI<random>.LOG`. Without this log file, it's not possible to determine why the installation failed.

**Comment by zkhiker** [comment]
[MSIa59f5.LOG](https://github.com/elastic/windows-installers/files/3454798/MSIa59f5.LOG)
Please check the MSI installation log file.

**Comment by zkhiker** [comment]
It failed for I selected one plugin.

**Comment by russcam** [comment]
The installation is failing because the installation of the analysis-smartcn plugin writes to stderr:

```
[0] installing analysis-smartcn installing analysis-smartcn
future versions of Elasticsearch will require Java 11; your Java version from  does not meet this requirement
-> Downloading analysis-smartcn from elastic
-> Installed analysis-smartcn
[0] installing analysis-smartcn installing analysis-smartcn
System.Exception: Execution failed (D:\tools\Elasticsearch\7.2.0\bin\elasticsearch-plugin.bat install analysis-smartcn --batch). ExitCode: 0 ErrorCheck: True ErrorOut:future versions of Elasticsearch will require Java 11; your Java version from  does not meet this requirement

   在 Elastic.Installer.Domain.Configuration.Plugin.PluginStateProviderBase.Install(Int32 pluginTicks, String installDirectory, String configDirectory, String plugin, String additionalArguments, IDictionary`2 environmentVariables)
   在 Elastic.Installer.Domain.Configuration.Plugin.ElasticsearchPluginStateProvider.Install(Int32 pluginTicks, String installDirectory, String configDirectory, String plugin, String additionalArguments, IDictionary`2 environmentVariables)
   在 Elastic.InstallerHosts.Elasticsearch.Tasks.Install.InstallPluginsTask.ExecuteTask()
   在 Elastic.InstallerHosts.SessionExtensions.Handle(Session session, Func`1 action)
```

This issue is addressed in #329. Until that PR is merged and makes its way into a release, the workaround is to perform the installation without installing the `analysis-smartcn` plugin, then install it after using the `elasticsearch-plugin.bat` file in the `%ES_HOME%\bin` directory, using

```bat
elasticsearch-plugin.bat install analysis-smartcn --batch
```
